### DISCUSSION TITLE
Manual deployment does not work, website not displayed

### DISCUSSION THREAD

**Discussion by cptalpdeniz** [getredash/redash#6625] [Q&A] [answered]
I followed the tutorial at https://github.com/getredash/redash/wiki/Local-development-setup using the Redash source code v10.1 however the website is neither showing up when I enter the domain name or the IP, what am I missing?

**Comment by cptalpdeniz** [comment]
This is the result of `docker ps --no-trunc`
![image](https://github.com/getredash/redash/assets/17345222/341a633c-d50b-4b3d-bd5a-5c0411886802)

**Reply by guidopetri** [reply]
Are you accessing the IP's port 5000?

**Reply by cptalpdeniz** [reply]
1- Can you explain what do you mean by that?
2- How do I check that?
3- If it's not setup how it should be, what should I do?

**Reply by guidopetri** [reply, marked as answer]
The port that the redash server is listening on is 5000, so you have to access the IP address on port 5000. I'd recommend reading more into port forwarding and how docker handles ports. You should be able to access your deployment by visiting `your.ip.addr.ess:5000`.

**Reply by cptalpdeniz** [reply]
"I'd recommend reading more into port forwarding and how docker handles ports"
from where though?

Also not sure why you selected this as the answer but I'm not able to reach my installation with that (ip:5000)

**Reply by cptalpdeniz** [reply]
I'm getting this error when I try to access it with the `ip:5000`
![image](https://github.com/getredash/redash/assets/17345222/a4112ca9-c074-4cda-801c-ec0346b48fb1)

```
The debugger caught an exception in your WSGI application. You can now look at the traceback which led to the error.

To switch between the interactive traceback and the plaintext one, you can click on the "Traceback" headline. From the text traceback you can also create a paste of it. For code execution mouse-over the frame you want to debug and click on the console icon on the right side.

You can execute arbitrary Python code in the stack frames and there are some extra helpers available for introspection:

    dump() shows all variables in the frame
    dump(obj) dumps all that's known about the object
```

**Reply by guidopetri** [reply]
>"I'd recommend reading more into port forwarding and how docker handles ports"
>from where though?

From the official docker documentation, or some kind of tutorial on docker, probably. It's up to you really, but given that that topic is much larger than redash (and has its own nuances) I would recommend doing that first.

> Also not sure why you selected this as the answer but I'm not able to reach my installation with that (ip:5000)

The screenshot you posted is indeed you reaching the redash container :) but it seems like something went wrong with the setup for the database, maybe. Did you run `make create_database`? What commands did you run specifically to set up redash?

**Reply by cptalpdeniz** [reply]
I ran the guide in this repo, in the Wiki section. Then, I ran the `make build` followed by `make up`. I will stop and remove all containers then try again executing `make create_database` first.
